pkg/services/alerting/conditions/query.go

```diff
@@ -113,7 +116,7 @@ func (c *QueryCondition) executeQuery(context *alerting.EvalContext, timeRange *
 	}
 
 	if err := bus.Dispatch(getDsInfo); err != nil {
-		return nil, fmt.Errorf("Could not find datasource %v", err)
+		return nil, fmt.Errorf("could not find datasource: %w", err)
```

Comment:
@aknuds1 This looks like something you updated - there was a conflict and I must have applied the change accidentally. This is fine right?

Reply:
@wbrowne Yes.